src/modules/webpack-publish.test.ts

```diff
@@ -0,0 +1,123 @@
+
+            const options: WebpackRestoreOptions = {
+                ci: undefined,
+                skipClean: undefined,
+                skipRestore: undefined,
+            };
+
+            // Act
+
+            WebpackPublish.restore(options);
+
+            // Assert
+            expect(nodeCISpy).not.toBeCalled();
+        });
+        test("when ci is false, then NodeCI is not called ", () => {
+            // Arrange
+            const options: WebpackRestoreOptions = {
+                ci: false,
+                skipClean: undefined,
+                skipRestore: undefined,
+            };
+
+            // Act
+            const nodeCISpy = jest.spyOn(NodeCI, "run");
+            WebpackPublish.restore(options);
+
+            // Assert
+            expect(nodeCISpy).not.toBeCalled();
+        });
+        test("when ci is true, then NodeCI is called ", () => {
+            // Arrange
+            const options: WebpackRestoreOptions = {
+                ci: true,
+                skipClean: undefined,
+                skipRestore: undefined,
+            };
+
+            // Act
+            const nodeCISpy = jest.spyOn(NodeCI, "run");
+            WebpackPublish.restore(options);
+
+            // Assert
+            expect(nodeCISpy).toBeCalled();
+        });
+        test("when skipClean is false, NodeClean is called", () => {
+            // Arrange
+            const options: WebpackRestoreOptions = {
+                ci: undefined,
+                skipClean: false,
+                skipRestore: undefined,
+            };
+
+            // Act
+            const nodeCISpy = jest.spyOn(NodeClean, "run");
+            WebpackPublish.restore(options);
+
+            // Assert
+            expect(nodeCISpy).toBeCalled();
+        });
+        test("when skipClean is true, NodeClean is not called", () => {
+            // Arrange
+            const options: WebpackRestoreOptions = {
+                ci: undefined,
+                skipClean: true,
+                skipRestore: false,
+            };
+
+            // Act
+            const nodeCISpy = jest.spyOn(NodeClean, "run");
+            WebpackPublish.restore(options);
+
+            // Assert
+            expect(nodeCISpy).not.toBeCalled();
+        });
+        test("when skipRestore is false, NodeRestore is called", () => {
+            // Arrange
+            const options: WebpackRestoreOptions = {
+                ci: undefined,
+                skipClean: false,
+                skipRestore: false,
+            };
+
+            // Act
+            const nodeCISpy = jest.spyOn(NodeRestore, "run");
+
+            WebpackPublish.restore(options);
+
+            // Assert
+            expect(nodeCISpy).toBeCalled();
+        });
+        test("when skipRestore is true, NodeRestore is not called", () => {
+            // Arrange
+            const options: WebpackRestoreOptions = {
+                ci: undefined,
+                skipClean: false,
```

Comment:
Note: in this case, it looks like this is testing "when skipClean is false and skipRestore is true"